__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const data = require("../db/data/test-data/");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("Path not found 404", () => {
+  test("returns 404 for path that doesnt exist", async () => {
+    const res = await request(app).get("/api/uhertiguer").expect(404);
+    const error = res.body;
+    expect(error.msg).toBe("Path Not Found");
+  });
+});
+
+describe("GET /api/topics", () => {
+  test("returns a 200 response code", async () => {
+    await request(app).get("/api/topics").expect(200);
+  });
```

Comment:
Nice thinking but since we test for a 200 in the next test it makes this one not really needed.